src/media/js/views/search.js

```diff
@@ -117,6 +117,7 @@ define('views/search',
         };
 
         var query = $('#search-q').val() || $('#search-q-desktop').val();
+        var query = query.replace( /  +/g, ' ' );
```

Comment:
Sorry for the delay, I missed the notification of the response.
This should work for the regex. @spasovski -- r?